Sources/XCResultKit/ActivityLogCommandInvocationSection.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ActivityLogCommandInvocationSection.swift
+//  XCResultKit
+//
+//  Created by Pierre Felgines on 05/10/2019.
+//
+
+import Foundation
+
+//- ActivityLogCommandInvocationSection
+//* Supertype: ActivityLogSection
+//* Kind: object
+//* Properties:
+//  + commandDetails: String
+//  + emittedOutput: String
+//  + exitCode: Int?
+
+public struct ActivityLogCommandInvocationSection: XCResultObject {
+    public let domainType: String
+    public let title: String
+    public let startTime: Date?
+    public let duration: Double
+    public let result: String?
+    public let subsections: [ActivityLogMajorSection]
+    public let unitTestSubsections: [ActivityLogUnitTestSection]
+    public let commandInvocationSubsections: [ActivityLogCommandInvocationSection]
+    public let targetBuildSubsections: [ActivityLogTargetBuildSection]
+    public let messages: [ActivityLogMessage]
+    public let resultMessages: [ActivityLogAnalyzerResultMessage]
+    public let warningMessage: [ActivityLogAnalyzerWarningMessage]
+
+    public let commandDetails: String
+    public let emittedOutput: String
```

Comment:
Looks like this needs to be optional. At least in the example you have for the unit tests. I've ran into several cases of this so far where the documentation doesn't line up exactly with actual output. But before I had suspected it would change after GM dropped. I suspect just a lag in the documentation getting updated for this.

Reply:
Good catch ! And I agree, documentation has not been updated with official release.